packages/nextjs/pages/builders/index.tsx

```diff
@@ -0,0 +1,58 @@
+import Link from "next/link";
+import type { NextPage } from "next";
+import { Address } from "~~/components/scaffold-eth";
+import { useScaffoldEventHistory } from "~~/hooks/scaffold-eth";
+
+const BuilderPage: NextPage = () => {
+  const {
+    data: eventData,
+    isLoading,
+    error,
+  } = useScaffoldEventHistory({
+    contractName: "BatchRegistry",
+    eventName: "CheckedIn",
+    fromBlock: 0n,
```

Comment:
Let's use the block that the contract was deployed on OP, so this is faster/efficient

Reply:
cool, good call :ok_hand: